frontend/src/pages/team/Settings/Danger.vue

```diff
@@ -62,13 +63,28 @@ export default {
             return this.user.admin
         },
         deleteActive () {
-            return this.applicationCount === 0
+            if (this.applicationCount === 0) {
+                return true
+            } else {
+                for (let i = 0; i < this.applicationList.count; i++) {
+                    if (this.applicationList.applications[i].instances.length !== 0) {
+                        return false
+                    }
+                }
+                return true
```

Comment:
this.applicationList.applications.some((application) => application.instances.length !== 0)
```suggestion
                return this.applicationList.applications.every((application) => application.instances.length === 0)
```